godot-ffi/src/lib.rs

```diff
@@ -447,6 +447,31 @@ pub fn is_main_thread() -> bool {
     }
 }
 
+/// Assign the current thread id to be the main thread.
+///
+/// This is required for platforms on which Godot runs the main loop on a different thread than the thread the library was loaded on.
+/// Android is one such platform.
+///
+/// # Safety
+///
+/// - must only be called after [`initialize`] has been called.
+pub unsafe fn make_main_thread() {
```

Comment:
```suggestion
pub unsafe fn discover_main_thread() {
```

Because the current name sounds like promoting the calling thread to the main thread, which isn't what this function does.